[python-babel] Add first-level keys for each numbering system in Locale.number_symbols

Manish7093 opened the pull request:
- Locale.number_symbols will now have first-level keys for each numbering system. Locale.number_symbols['decimal'] is now Locale.number_symbols['latn']['decimal'].
- https://github.com/python-babel/babel/releases/tag/v2.14.0

Manish7093 committed:
Add first-level keys for each numbering system in Locale.number_symbols

- Locale.number_symbols will now have first-level keys for each numbering system. Locale.number_symbols['decimal'] is now Locale.number_symbols['latn']['decimal'].
- https://github.com/python-babel/babel/releases/tag/v2.14.0

imhemish commented:
yes, after building, ibus-stt failed to launch because of key error due to this. I put the ['latn'] and it worked!

Manish7093 committed:
Fix redownload issue in STT setup tool

Manish7093 committed:
Merge pull request #1 from Manish7093/fix_redownload_issue

Fix redownload issue in STT setup tool

Manish7093 committed:
Move required models out of obsolete list

Manish7093 committed:
Move required models out of obsolete list

Added "vosk-model-en-us-daanzu-20200905", "vosk-model-en-us-daanzu-20200905-lgraph" models to Available Model list

Manish7093 committed:
Correct chain up for do_startup

Manish7093 committed:
Add WhisperCpp Support (#4)

Manish7093 committed:
Limit Whisper n_threads for better resource efficiency (#5)

Manish7093 committed:
Supress Whisper's non-speech tokens

Manish7093 committed:
Add missing re import

SidShetye commented:
@PhilippeRo - will you be merging this? @Manish7093 has additional changes at https://github.com/Manish7093/IBus-Speech-To-Text/tree/main which would be good to review and merge. Or have you discontinued this project?

Manish7093 committed:
Add support for Quantized Whisper models (#8)

Manish7093 committed:
engine: Replace AdwPreferencesPage with GtkBox for categorypage

Manish7093 committed:
Add WhisperCpp model description (#10)

Manish7093 committed:
improve WhisperCpp streaming pipeline

Manish7093 committed:
engine/sttconfigdialog: redesign stt configdialog ui

Manish7093 committed:
feature: add energy based vad(Voice Activity Detection) support

Manish7093 committed:
engine/sttconfigdialog:persist selected locale and correctly detect installed models

daym committed:
Add onnx-asr backend.

Manish7093 committed:
Add moonshine-voice backend

Manish7093 committed:
engine/sttconfigdialog: fix formatting file msg for non-vosk backend

Manish7093 committed:
ui: update setup tool label for voice commands

Manish7093 committed:
engine/sttengine: update project links and version

Manish7093 committed:
engine/moonshine: support the moonshine-voice v0.1.0

Manish7093 committed:
engine: notify user when a backend's python module is missing
